src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -243,7 +250,11 @@
   [server :- Server
    config :- config/WebserverConnector]
   (let [request-size (:request-header-max-size config)]
-    (doto (ServerConnector. server (connection-factory request-size))
+    (doto (ServerConnector. server
+                            nil nil nil
```

Comment:
I see a constructor that allows you to specify an acceptor count, selector count, and `SslContextFactory`.  I'm not seeing one that allows you to specify acceptor count, selector count, and a ConnectionFactory other than the one where you have to also pass in the `Executor`, `Scheduler`, and `ByteBufferPool`, though.